global-configuration/src/main/resources/archetype-resources/pom.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.jenkins-ci.plugins</groupId>
+        <artifactId>plugin</artifactId>
+        <version>2.36</version>
+        <relativePath />
+    </parent>
+    <artifactId>${artifactId}</artifactId>
+    <version>${version}</version>
+    <packaging>hpi</packaging>
+    <properties>
+        <jenkins.version>2.60.3</jenkins.version>
+        <java.level>8</java.level>
```

Comment:
Need at least 2.19.x to pick up https://github.com/jenkinsci/jenkins/pull/2509, but why not make `RestartableJenkinsRule` look pretty while we are here?

Reply:
Not going to complain about encouraging new plugins to start on Java 8, that's for sure.